lib/entities/organization-membership.ts

```diff
@@ -0,0 +1,127 @@
+import cloneDeep from 'lodash/cloneDeep'
+import { freezeSys, toPlainObject } from 'contentful-sdk-core'
+import enhanceWithMethods from '../enhance-with-methods'
+import errorHandler from '../error-handler'
+import { createDeleteEntity } from '../instance-actions'
+import { AxiosInstance } from 'axios'
+import { MetaSysProps, QueryOptions, DefaultElements, CollectionProp } from '../common-types'
+
+export interface Options {
```

Comment:
I wouldn't create a special type for this and rather use `QueryOptions` in place

Reply:
@suevalov thx for feedback! yeah, make much more sense. It's updated, please have a look if this is how we would like to solve it.